pywt/data/_readers.py

```diff
@@ -137,3 +137,43 @@ def ecg():
     fname = os.path.join(os.path.dirname(__file__), 'ecg.npy')
     ecg = np.load(fname)
     return ecg
+
+
+def nino():
+    """
+    NINO3 sea surface temperature 1871-1996 (in C; from the UKMO GISST2.3):
```

Comment:
More importantly, looks like this dataset doesn't have an appropriate license: http://www.metoffice.gov.uk/hadobs/gisst/terms_and_conditions.html

Or is there a public domain or more liberally licensed version somewhere else? http://paos.colorado.edu/research/wavelets/wave_python/ doesn't really say anything about that.